src/mesh/parallel/shiftedmetric.cxx

```diff
@@ -136,18 +136,47 @@ void ShiftedMetric::cachePhases() {
  * and Y is then field aligned.
  */
 const Field3D ShiftedMetric::toFieldAligned(const Field3D& f, const REGION region) {
-  return shiftZ(f, toAlignedPhs, region);
+  switch (f.getDirectionY()) {
+  case (DIRECTION::YOrthogonal):
+    return shiftZ(f, toAlignedPhs, DIRECTION::YOrthogonal, region);
+  case (DIRECTION::YAligned):
+    // f is already in field-aligned coordinates
+    return f;
+  default:
+#if CHECK > 0
+    throw BoutException("Unrecognized y-direction type for Field3D passed to "
```

Comment:
Can we just always do this? This is essentially free, so I don't see a downside to removing the `#if CHECK`